doc/running/index.rst

```diff
@@ -201,3 +201,18 @@ This will generate, compile, build, and install the code for a set of specified
      - (Optional) Enable development mode: code generation is attempted even for models that contain errors, and extra information is rendered in the generated code. Default is OFF.
    * - ``--codegen_opts``
      - (Optional) Path to a JSON file containing additional options for the target platform code generator. A list of available options can be found under the section "Code generation options" for your intended target platform on the page :ref:`Running NESTML`.
+
+NEST Desktop target
```

Comment:
Could you instead add a button like for Python-standalone and SpiNNaker under "Supported target platforms" (https://nestml.readthedocs.io/en/latest/running/index.html#supported-target-platforms)?